VirtoCommerce.Platform.Web/Hangfire/HangfireAuthorizationHandler.cs

```diff
@@ -1,23 +1,33 @@
-using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 using Hangfire.Dashboard;
-using Microsoft.AspNetCore.Identity;
 using VirtoCommerce.Platform.Core;
-using VirtoCommerce.Platform.Core.Security;
+using VirtoCommerce.Platform.Security.Services;
 
 namespace VirtoCommerce.Platform.Web.Hangfire
 {
     public class HangfireAuthorizationHandler : IDashboardAuthorizationFilter
     {
+        private readonly LimitedPermissionsHandler _limitedPermissionsHandler;
```

Comment:
I think we can leave this handler as it. We don't need this extra checks here, because it won't violated the limited permissions logic.

Reply:
I continue to be sure that this verification is absolutely is redundant. Since adding BackgroundJobsManage permissions to limited cookies permissions in the settings will have the same effect that we will achieve simply by excluding this additional check here.